test/e2e/common/node/pod_resize.go

```diff
@@ -1215,7 +1215,7 @@ var _ = SIGDescribe("Pod InPlace Resize Container", feature.InPlacePodVerticalSc
 	ginkgo.BeforeEach(func(ctx context.Context) {
 		node, err := e2enode.GetRandomReadySchedulableNode(ctx, f.ClientSet)
 		framework.ExpectNoError(err)
-		if framework.NodeOSDistroIs("windows") || e2enode.IsARM64(node) {
+		if e2enode.IsARM64(node) {
```

Comment:
Since these tests spin up Linux pods, they won't enable IPPVS testing on Windows.  Running them on Windows would likely require significant refactoring.